chaosazure/__init__.py

```diff
@@ -98,3 +154,20 @@ def __load_exported_activities() -> List[DiscoveredActivities]:
     activities.extend(discover_actions("chaosazure.webapp.actions"))
     activities.extend(discover_probes("chaosazure.webapp.probes"))
     return activities
+
+
+def __get_cloud_env_by_name(cloud=None):
+    try:
+        if cloud is None:
+            return msrestazure.azure_cloud.AZURE_PUBLIC_CLOUD
+        elif cloud.strip() == "AZURE_CHINA_CLOUD":
```

Comment:
would be better to do the `.strip()` once only at the beginning of the function (when it isn't None that is).